package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "whirli-client",
-  "version": "0.0.7",
+  "version": "0.0.8",
```

Comment:
Please only change this version on the PR from develop to master, when the code gets released